src/Pages/URLShortener/URLShortener.js

```diff
@@ -33,9 +35,17 @@ export default function URLShortenerPage(props) {
    * Cleezy page is disabled by default since you have to run the Cleezy server
    * separately. To enable, go to config.json and set ENABLED under Cleezy to true
    */
-  async function getCleezyUrls(page, searchQuery) {
+  async function getCleezyUrls(page, searchQuery, currentSortColumn, currentSortOrder) {
     setLoading(true);
-    const urlsFromDb = await getAllUrls(props.user.token, page, searchQuery);
+    const sortColumn = currentSortOrder === 'none' ? 'created_at' : currentSortColumn;
+    const sortOrder = currentSortOrder === 'none' ? 'DESC' : currentSortOrder;
```

Comment:
see the null comment then we can do

```suggestion
    const sortColumn = currentSortColumn ?? 'created_at';
    const sortOrder = currentSortOrder ?? 'DESC';
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing